abs.yaml

```
@@ -0,0 +1,2 @@
+upload_channels:
```

```suggestion
channels:
  - sfe1ed40
upload_channels:
```
To get access to opencv-python-headless.

---

`upload_channels` should work also for sourcing the deps, I'll try first without `channels`

---

it works